src/components/modules/dragNDrop.ts

```diff
@@ -119,4 +217,47 @@ export default class DragNDrop extends Module {
   private processDragOver(dragEvent: DragEvent): void {
     dragEvent.preventDefault();
   }
+
+  /**
+   * Create drag image for drag-n-drop.
+   *
+   * @param blocks {Block[]} - blocks to create drag image for.
+   * @returns {HTMLElement} - drag image.
+   */
+  private createDragImage(blocks: Block[]): HTMLElement {
+    const { BlockManager } = this.Editor;
+
+    /**
+     * If we are dragging only one block, return its content.
+     */
+    if (blocks.length == 1) {
+      const block = BlockManager.currentBlock;
+
+      return block.holder.querySelector(`.${Block.CSS.content}`);
+    }
+
+    /**
+     * If we are dragging multiple blocks, create a drag image with all blocks content.
+     */
+    const dragImage: HTMLElement = document.createElement('div');
+
+    dragImage.id = `drag-image-${_.generateId()}`;
+    dragImage.style.position = 'absolute';
+    dragImage.style.top = '-1000px';
+    blocks.forEach(block => {
+      const blockContent = block.holder.querySelector(`.${Block.CSS.content}`).cloneNode(true);
+
+      dragImage.appendChild(blockContent);
+    });
+    document.body.appendChild(dragImage);
+
+    return dragImage;
+  }
+
+  /**
+   * Remove drag image from DOM.
+   */
+  private removeDragImage(): void {
+    document.querySelector('[id^="drag-image-"]')?.remove();
```

Comment:
What if you are dragging only one block? I don't see where you add this id for an element in this case. Will this element be removed by this method?